pkg/skaffold/build/kaniko/sources/localdir.go

```diff
@@ -105,12 +105,13 @@ func (g *LocalDir) ModifyPod(ctx context.Context, p *v1.Pod) error {
 		return errors.Wrap(err, "waiting for pod to initialize")
 	}
 	// Copy over the buildcontext tarball into the init container
-	copy := exec.CommandContext(ctx, "kubectl", "cp", g.tarPath, fmt.Sprintf("%s:/%s", p.Name, g.tarPath), "-c", initContainer, "-n", p.Namespace)
+	tarCopyPath := fmt.Sprintf("/tmp/%s", filepath.Base(g.tarPath))
+	copy := exec.CommandContext(ctx, "kubectl", "cp", g.tarPath, fmt.Sprintf("%s:%s", p.Name, tarCopyPath), "-c", initContainer, "-n", p.Namespace)
```

Comment:
I don't think we need the `tarCopyPath` variable, since copying to `g.tarPath` should accomplish the same thing, right?

Reply:
@priyawadhwa It doesn't work in windows unfortunately. That is the reason I had to use `tarCopyPath` .

`g.tarPath` = `os.TempDir()` in windows ~ C:\Users\<user-id>\AppData\Local\Temp